pkg/controller/pv.go

```diff
@@ -50,6 +50,7 @@ type ReconcilePV struct {
 	kubeClient     kubeClientSet.Interface
 	// retryMap allows volumeAttachment controller to retry Get operation for AzVolume in case the CRI has not been created yet
 	retryMap              sync.Map
+	retryLocks            sync.Map
```

Comment:
Instead of two different maps which have the same keys (and incur double lookups), consider capturing the `sync.Mutex` and `wait.Backup` in a common struct, e.g.
```go
type retryInfo struct {
    mutex   sync.Mutex
    backoff wait.Backoff
}
```

Reply:
It seems like you could also encode the details of getting the requeue time in a method, e.g.
```go
func (r *retryInfo) nextRequeue() {
    ...
}
```

Reply:
This struct should ideally go in its own source file since it is shared with the `VolumeAttachment` controller.

Reply:
I was considering adding the common struct to `common.go`. Should I create a new dedicated source file for this?

Reply:
`common.go` should be fine.